java/jdbc/src/main/java/com/flipkart/vitess/jdbc/VitessJDBCUrl.java

```diff
@@ -102,8 +109,8 @@ public VitessJDBCUrl(String url, Properties info) throws SQLException {
         }
 
         this.tabletType = getTabletType(tabletType);
-
         this.executeType = info.getProperty(Constants.Property.EXECUTE_TYPE);
+        this.ssl = "true".equals(info.getProperty(Constants.Property.USE_SSL));
```

Comment:
Do you have any objection to the property **_name_** remaining case-sensitive?  (i.e. `useSSL` is valid, `usessl` is invalid)

I'm happy to make both the name and value case-insensitive.  But it will require a bit more code, and there are several other camel-cased optional properties to come shortly (e.g. `keyStore`, `keyStorePassword`, etc).  So I just want to make sure I take the desired approach in the first pass.